users/tests.py

```diff
@@ -137,10 +139,264 @@ def test_whoami(self):
+        url = reverse('token-obtain-pair', kwargs={'version': 'v1'})
+        auth = self.client.post(url, data=json.dumps(self._user), content_type='application/json')
+        fake_header = b'Bearer ' + json.loads(auth.content)['access'].encode()
+
+        password = 'SuperSecret'
+        data = {
+            'password': password,
+            'retype_password': password,
+        }
+        url = reverse('password-update', kwargs={'version': 'v1'})
+        response = self.client.post(url, data=json.dumps(data), content_type='application/json',
+                                    HTTP_AUTHORIZATION=fake_header)
+
+        self.assertEqual(response.content, b'{"old_password": ["This field may not be blank."]}')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+        data = {
+            'old_password': self._user['password'],
+            'retype_password': password,
+        }
+        url = reverse('password-update', kwargs={'version': 'v1'})
+        response = self.client.post(url, data=json.dumps(data), content_type='application/json',
+                                    HTTP_AUTHORIZATION=fake_header)
+
+        self.assertEqual(response.content, b'{"password": ["This field may not be blank."]}')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+        data = {
+            'old_password': self._user['password'],
+            'password': password,
+        }
+        response_content = (b'{"password": ["Passwords mismatch"], '
+                            b'"retype_password": ["This field may not be blank."]}')
+        url = reverse('password-update', kwargs={'version': 'v1'})
+        response = self.client.post(url, data=json.dumps(data), content_type='application/json',
+                                    HTTP_AUTHORIZATION=fake_header)
+
+        self.assertEqual(response.content, response_content)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_password_update_with_incorrect_old_password(self):
+        url = reverse('token-obtain-pair', kwargs={'version': 'v1'})
+        auth = self.client.post(url, data=json.dumps(self._user), content_type='application/json')
+        fake_header = b'Bearer ' + json.loads(auth.content)['access'].encode()
+
+        password = 'SuperSecret'
+        data = {
+            'old_password': self._user['password'] * 2,
+            'password': password,
+            'retype_password': password,
+        }
+        url = reverse('password-update', kwargs={'version': 'v1'})
+        response = self.client.post(url, data=json.dumps(data), content_type='application/json',
+                                    HTTP_AUTHORIZATION=fake_header)
+
+        self.assertEqual(response.content, b'{"old_password": ["Incorrect old password"]}')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_password_update_with_mismatch_passwords(self):
```

Comment:
Let's
* rename the method into `test_password_update_when_passwords_mismatch`
* use the following docsting for the method:
   `""""Tests if it's not possible to update the current user password when the old and new passwords mismatch. """`